From: =?utf-8?b?Ik1hcmNvIFRyZXZpc2FuIChUcmV2acOxbyki?= <mail@3v1n0.net>
Date: Thu, 17 Jun 2021 13:51:47 +0200
Subject: meson: Only generate typelib and install win32 files on windows

There's no need to expose them in other platforms, but we do.

So only include the win32-1.0.gir in windows.

Origin: https://gitlab.gnome.org/GNOME/gobject-introspection/-/merge_requests/280
---
 gir/meson.build | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/gir/meson.build b/gir/meson.build
index 557e551..87cb5ca 100644
--- a/gir/meson.build
+++ b/gir/meson.build
@@ -35,9 +35,12 @@ gir_files = [
   'xlib-2.0.gir',
   'xfixes-4.0.gir',
   'xrandr-1.3.gir',
-  'win32-1.0.gir',
 ]
 
+if host_system == 'windows'
+  gir_files += 'win32-1.0.gir'
+endif
+
 typelibdir = join_paths(get_option('libdir'), 'girepository-1.0')
 install_data(gir_files, install_dir: girdir)
 
